internal/characters/hutao/frames.go

```diff
@@ -2,36 +2,117 @@ package hutao
 
 import "github.com/genshinsim/gcsim/pkg/core"
 
-/**
-[11:32 PM] sakuno | yanfei is my new maid: @gimmeabreak
-https://www.youtube.com/watch?v=3aCiH2U4BjY
-
-framecounts for 7 attempts of N2CJ (no hitlag):
-83, 85, 88, 89, 77, 82, 84
-
-first 3 not from the uploaded recording (as a n1cd player i cud barely pull it off :monkaS: )
-YouTube
-**/
-
-//var normalFrames = []int{13, 16, 25, 36, 44, 39}               // from kqm lib
-var normalFrames = []int{10, 13, 22, 33, 41, 36} // from kqm lib, -3 for hit lag
```

Comment:
I think you should keep frames in here actually instead of moving it into `abil.go`. I know other files have done otherwise but this is a more sensible grouping.